frontend/public/index.html

```diff
@@ -64,5 +64,29 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script>
+      if ('%NODE_ENV%' === 'production' && "serviceWorker" in navigator) {
+        window.addEventListener("load", function () {
```

Comment:
You can remove logging now and add JS arrow methods to make this much more concise:
  window.onload = () => {
    navigator.serviceWorker.register('./serviceWorker.js');
  }